files/mods.json

```diff
@@ -164,6 +164,29 @@
         "packages": ["potioneffects", "redaction", "polyfrostkeystrokes"],
         "hash": "8e3980e5abaf1c5c5adb4dae6f3ed9f4"
     },
+    {
+        "id": "quickjoin",
+        "display": "Quickjoin",
+        "creator": "QWERTZ_EXE",
+        "description": "Quickly join any game on Hypixel!",
+        "actions": [
+            {
+                "icon": "github.png",
+                "text": "GitHub",
+                "link": "https://github.com/QWERTZexe/Quickjoin"
+            },
+            {
+                "icon": "modrinth.png",
+                "text": "Modrinth",
+                "link": "https://modrinth.com/mod/quickjoin"
+            }
+        ],
+        "categories": ["Other"],
```

Comment:
Pretty sure this isn't a valid category

Reply:
idk, but here it shows "Other"
![image](https://github.com/SkyblockClient/SkyblockClient-REPO/assets/109920359/eb285854-f637-4c15-a623-fa5624fed670)